src/utils/clack-utils.ts

```diff
@@ -696,13 +706,16 @@ export async function runPrettierIfInstalled(): Promise<void> {
 
     try {
       await new Promise<void>((resolve, reject) => {
-        childProcess.exec('npx prettier --write .', (err) => {
-          if (err) {
-            reject(err);
-          } else {
-            resolve();
-          }
-        });
+        childProcess.exec(
+          `npx prettier --write ${prettierSupportsCache ? '--cache' : ''} .`,
```

Comment:
Does this work if users themselves didn't use `--cache` before? Reading the [documentation](https://prettier.io/docs/en/cli.html#--cache), I'm not sure because it sounds like a cache already needs to be populated so that a file is skipped. 

I'm not really worried about the performance aspect but rather that we format files that users (for whatever reason) haven't formatted before or don't want formatted. 

We already have some logic around detecting new and changed files via git at the beginning of the wizard. Maybe we could re-use that?